pyuvdata/calfits.py

```diff
@@ -257,6 +275,7 @@ def write_calfits(self, filename, run_check=True,
 
         prihdr['CTYPE3'] = ('TIME', 'Time axis.')
         prihdr['CUNIT3'] = ('JD', 'Time in julian date format')
+        prihdr['CRPIX3'] = 1
         prihdr['CRVAL3'] = self.time_array[0]
```

Comment:
Oh, yes it should be since it is it's own axis. I don't think that's checked, but I can add a check. Good catch!